_includes/selector.html

```diff
@@ -673,8 +702,7 @@
             },
             disableUnsupportedCuda(cuda_version) {
                 var isDisabled = false;
-                if (this.active_additional_packages.includes("TensorFlow") && (cuda_version !== "12.0")) isDisabled = true;
-                if (this.active_method === "Docker" && cuda_version < "11.8") isDisabled = true;
+                if (this.active_additional_packages.includes("TensorFlow") && (!cuda_version.startsWith("12"))) isDisabled = true;
                 if (this.active_release === "Stable" && cuda_version === "12.5") isDisabled = true;
```

Comment:
Curious if we could use a variable for `12.5` here as well

Reply:
Could we leave this for a later PR? I think this ties in with the [above thread](https://github.com/rapidsai/docs/pull/524#discussion_r1688249043), where I struggled to get a "structured" JSON format to work.